.travis.yml

```diff
@@ -22,19 +22,19 @@ before_install:
   - rvm @global do yes | gem install bundler -v '< 2'
 matrix:
   exclude:
-    - rvm: 2.4.5
+    - rvm: 2.4.9
       gemfile: gemfiles/rails_6.0.gemfile
     - rvm: 2.3.8
       gemfile: gemfiles/rails_6.0.gemfile
     - rvm: 2.2.10
       gemfile: gemfiles/rails_6.0.gemfile
     - rvm: ruby-head
       gemfile: gemfiles/rails_3.2.gemfile
-    - rvm: 2.6.2
+    - rvm: 2.6.5
       gemfile: gemfiles/rails_3.2.gemfile
-    - rvm: 2.5.5
+    - rvm: 2.5.7
       gemfile: gemfiles/rails_3.2.gemfile
-    - rvm: 2.4.5
+    - rvm: 2.4.9
       gemfile: gemfiles/rails_3.2.gemfile
   allow_failures:
-    - rvm: ruby-head
\ No newline at end of file
+    - rvm: ruby-head
```

Comment:
Please do not remove this line 🙏 

New lines at the end of the file are the standard in POSIX

Reply:
Au contraire, it was lacking a new line and I added one.

Reply:
ooh! you're right, that tooltip confused me

![Captura de pantalla 2019-10-09 a las 22 10 36](https://user-images.githubusercontent.com/576701/66516606-aea37580-eae1-11e9-95a3-b3915c4aebfe.png)